content/src/content/jcr_root/apps/core/wcm/components/container/v1/container/clientlibs/design/.content.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we usually call the folder `editor` not `design` in container/v1/container/clientlibs/design/.content.xml